CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # nf-core/tools: Changelog
 
+## v3.3.2dev
+
+### Template
+
+- Avoid overriding `NFT_DIFF` and `NFT_DIFF_ARGS` in `nf-test` action ([#3606](https://github.com/nf-core/tools/pull/3606)) and ([#3619](https://github.com/nf-core/tools/pull/3619))
+- fix nf-test scope to ignore nf-core module/swf tests ([#3609](https://github.com/nf-core/tools/pull/3609))
+- write github.run_id on pipeline template ([#3637](https://github.com/nf-core/tools/pull/3637))
+- Bump nf-schema to `2.4.2` ([#3533](https://github.com/nf-core/tools/pull/3533))
+- Bump the minimal Nextflow version to `24.10.5` ([#3533](https://github.com/nf-core/tools/pull/3533), [#3667](https://github.com/nf-core/tools/pull/3667))
+- CI - Only trigger nf-test action on pull_request ([#3628](https://github.com/nf-core/tools/pull/3628))
+- Fix link to nf-test GHA in README.md ([#3630](https://github.com/nf-core/tools/pull/3630))
+- Add accelerator directive for GPU-enabled processes ([#3632](https://github.com/nf-core/tools/pull/3632))
+- Update dependency prettier to v3.6.0 ([#3641](https://github.com/nf-core/tools/pull/3641)) and 3.6.2 ([#3646](https://github.com/nf-core/tools/pull/3646))
+
+### Linting
+
+- Fix linting of nf-test files content ([#3603](https://github.com/nf-core/tools/pull/3603))
+
+### Modules
+
+- Remove args stub from module template to satisfy language server ([#3403](https://github.com/nf-core/tools/pull/3403))
+- Fix modules meta.yml file structure ([#3532](https://github.com/nf-core/tools/pull/3532))
+- Fix wrong key when updating module outputs ([#3665](https://github.com/nf-core/tools/pull/3665))
+
+### Subworkflows
+
+### General
+
+- Add conda cache dir ([#3610](https://github.com/nf-core/tools/pull/3610))
+- update id of ruff hook in pre-commit config ([#3621](https://github.com/nf-core/tools/pull/3621))
```

Comment:
I usually like to move the version updates to a new section and remove duplicates. So if something has been updated multiple times and has multiple PRs linked, I keep only the most recent one

Reply:
fixed in [`ab3c3ce` (#3666)](https://github.com/nf-core/tools/pull/3666/commits/ab3c3ce90ccd42c0235ab0d518c2cdbdf64849aa)